cmd/jb-server/main.go

```diff
@@ -57,6 +57,24 @@ func main() {
 			Usage:  "path to compute cluster that VMs will be booted in",
 			EnvVar: "JUPITER_BRAIN_VSPHERE_CLUSTER_PATH",
 		},
+		cli.IntFlag{
+			Name:   "vsphere-concurrent-read-operations",
+			Usage:  "number of concurrent read vSphere operations",
```

Comment:
What about changing it to "number of concurrent fetch and list operations", and the others to "number of concurrent start/terminate operations?